corehq/motech/openmrs/repeaters.py

```diff
@@ -250,13 +250,18 @@ def send_request(self, repeat_record, payload):
             form_question_values=get_form_question_values(payload),
         )
 
-        return send_openmrs_data(
-            self.requests,
-            self.domain,
-            payload,
-            self.openmrs_config,
-            case_trigger_infos,
-        )
+        try:
+            response = send_openmrs_data(
+                self.requests,
+                self.domain,
+                payload,
+                self.openmrs_config,
+                case_trigger_infos,
+            )
+        except Exception as err:
+            self.requests.notify_exception(str(err))
+            return OpenmrsResponse(400, 'Bad Request', pformat_json(str(err)))
```

Comment:
Is this accurate?  Looking at `send_openmrs_data`, it seems like it handles network errors already, such that anything else I'd expect to be a novel error.

Reply:
Yes, this is for programmatic errors.

I think if `send_openmrs_data()` gets a RequestException when it's sending a case to OpenMRS it should move on to the next case. Once it has processed all the cases in the payload, it returns a mock HTTP response (`OpenmrsResponse`) so that the Repeat Records report will render it nicely in a dialog box.

But `send_openmrs_data()` doesn't catch programmatic errors. I added this `try ... except` block when I was getting an AttributeError. My thinking here was that `send_openmrs_data()` should not move on to the next case -- the error should be raised up to here. Catching it here is to email admins a notification, and allow the Repeat Records report to show the error in the same dialog box as it does request errors.

Let me know if you think I'm thinking about this the wrong way. I'd be happy to amend in a follow-up PR. But I'd like to merge this fix and deploy as soon as possible.